src/seml/experiment/config.py

```diff
@@ -733,7 +747,7 @@ def _convert_value(value):
         return value
 
 
-def convert_values(val):
+def convert_values(val: Any):
```

Comment:
If one wants to be super correct, one could provide overloading definitions to solve that if the input is of dict[K, V], the output will be of dict[K, Any]. (which avoids the cast in L.769?)